pages/index.js

```diff
@@ -1,12 +1,20 @@
 import TopicBox from "../components/organisms/TopicBox";
 import Layout from "../components/layout";
 import userSwr from "swr";
-import TitleSection from "../components/molecules/TitleSection";
+import { useContext } from "react";
+import { LanguageContext } from "../context/languageProvider";
 import Breadcrumb from "../components/molecules/Breadcrumb";
 
+import en from "../locales/en";
+import fr from "../locales/fr";
+
 const fetcher = (url) => fetch(url).then((res) => res.json());
 
 export default function Home() {
+  const { items } = useContext(LanguageContext);
+  const language = items.language;
+  const t = language === "en" ? en : fr;
```

Comment:
Is this code that we can generalize somehow? Just asking the question. It's fine as it is since we only have 2 pages.